controller/api/v1alpha1/servicedeployment_types.go.go

```diff
@@ -31,6 +32,8 @@ type ServiceHelm struct {
 	// +optional
 	ValuesRef *corev1.ConfigMapKeySelector `json:"values,omitempty"`
 	// +optional
+	ValuesRaw *runtime.RawExtension `json:"valuesRaw,omitempty"`
```

Comment:
could we just name this `values`?  Would be more what I'd expect as a user.

Reply:
Sure, but it will be a breaking change because `values` is already used above for config map ref. Existing CRDs with old spec will have to be updated first.

Reply:
Oh I see why, I wouldn't worry about that breaking change, it's almost not being used explicitly, certainly not by high priority users